datacapture/src/main/java/com/google/android/fhir/datacapture/Configuration.kt

```diff
@@ -16,23 +16,21 @@
 
 package com.google.android.fhir.datacapture
 
-import android.app.Application
 import org.hl7.fhir.r4.context.SimpleWorkerContext
 import org.hl7.fhir.r4.model.Coding
 import org.hl7.fhir.r4.model.StructureMap
 import org.hl7.fhir.utilities.npm.NpmPackage
 
 /**
- * The clients may use [DataCaptureConfig] to provide [ExternalAnswerValueSetResolver] and
- * [NpmPackage] to the library. The clients should set the configuration in [Application.onCreate]
- * as it would retain it across configuration changes.
+ * The clients may provide the [Configuration] for the DataCapture library using
+ * [DataCapture.initialize].
  */
-object DataCaptureConfig {
+data class Configuration(
```

Comment:
I think this name is just a little too generic, what about
```suggestion
data class DataCaptureConfig(
```